package.json

```diff
@@ -1,18 +1,36 @@
 {
   "name": "react-popper",
-  "version": "0.8.3",
+  "version": "0.9.0",
   "description": "React wrapper around PopperJS.",
+  "license": "MIT",
+  "author":
+    "Travis Arnold <travis@souporserious.com> (http://souporserious.com)",
+  "homepage": "https://github.com/souporserious/react-popper",
   "main": "dist/react-popper.js",
+  "umd:main": "dist/react-popper.umd.js",
   "module": "lib/react-popper.js",
   "types": "react-popper.d.ts",
-  "files": [
-    "dist",
-    "lib",
-    "react-popper.d.ts"
-  ],
-  "license": "MIT",
-  "author": "Travis Arnold <travis@souporserious.com> (http://souporserious.com)",
-  "homepage": "https://github.com/souporserious/react-popper",
+  "files": ["dist", "lib"],
+  "scripts": {
+    "build":
+      "npm run build:clean && npm run build:es && npm run build:umd && npm run build:cjs && npm run build:umd-min && npm run build:cjs-min",
+    "build:clean": "rm -rf dist/ && rm -rf lib/",
+    "build:es":
+      "babel src --out-dir lib && mv lib/index.js lib/react-popper.js",
+    "build:umd":
+      "rollup -c --output.format umd --output.name 'react-popper' --output.file dist/react-popper.umd.js",
+    "build:cjs":
+      "rollup -c --output.format cjs --output.name 'react-popper' --output.file dist/react-popper.js",
+    "build:umd-min":
+      "MINIFY=true rollup -c --output.format umd --output.name 'react-popper' --output.file dist/react-popper.umd.min.js",
+    "build:cjs-min":
+      "MINIFY=true rollup -c --output.format cjs --output.name 'react-popper' --output.file dist/react-popper.min.js",
+    "demo": "parcel --out-dir demo/dist demo/index.html",
+    "prepare": "npm run build",
```

Comment:
Correct me if I am wrong, but `perpare` will run when users install this library as a dependency (as well as before the package is published). Did you intend to have it build every time it is installed?
`prepublishOnly` would run the build _only_ prior to publishing (not on install).

Reply:
npm sucks, there's not a real solution to this...

Reply:
I agree, 

> Since `npm@1.1.71`, the npm CLI has run the `prepublish` script for both `npm publish` and `npm install`, because it's a convenient way to prepare a package for use. It has also turned out to be, in practice, [very confusing](https://github.com/npm/npm/issues/10074). As of `npm@4.0.0`, a new event has been introduced, `prepare`, that preserves this existing behavior. A new event, `prepublishOnly` has been added as a transitional strategy to allow users to avoid the confusing behavior of existing npm versions and only run on `npm publish` (for instance, running the tests one last time to ensure they're in good shape).